spring-cloud-contract-wiremock/src/test/resources/application-customfoo.yml

```diff
@@ -0,0 +1,7 @@
+wiremock:
+  server:
+    port: 0
+    stubs:
+      - "file:src/test/resources/custom-from-yml/mappings"
```

Comment:
Again, pointing to these custom dirs assures us that the user overrides here in the yaml are being used.